modules/common/development/debug-tools.nix

```
@@ -37,10 +37,16 @@ in
           # Performance testing
           speedtest-cli
           iperf
+
+          # Let's have this fixed version according to kernel.
+          # It would be possible to select also latest producing currently (12-3-2024) perf version 6.6.7
+          # linuxPackages_latest.perf
+          linuxKernel.packages.linux_6_1.perf
```

```suggestion
          config.boot.kernelPackages.perf
```

This automatically pulls in perf for the specific kernel the system is built with.

---

Thanks. Changed to that.